micromamba/tests/test_remove.py

```diff
@@ -59,12 +56,9 @@ def test_remove_orphaned(tmp_home, tmp_root_prefix, tmp_xtensor_env, tmp_env_nam
     keys = {"dry_run", "success", "prefix", "actions"}
     assert keys.issubset(set(res.keys()))
     assert res["success"]
-    # TODO: find a better use case so we can revert to len(env_pkgs) instead
-    # of magic number
-    # assert len(res["actions"]["UNLINK"]) == len(env_pkgs) + (
-    assert len(res["actions"]["UNLINK"]) == 3 + (
-        1 if helpers.dry_run_tests == helpers.DryRun.DRY else 0
-    )
+
+    assert len(res["actions"]["UNLINK"]) == len(env_pkgs)
```

Comment:
Actually this line was not alone, I think we still need the 2 lines that you removed:
```suggestion
    assert len(res["actions"]["UNLINK"]) == len(env_pkgs) + (
          1 if helpers.dry_run_tests == helpers.DryRun.DRY else 0
    )
```
But we can wait and see the test status.

Reply:
This previous value does not work. I just reverted the last commit.

Reply:
Yes because it was still missing the two lines below (just replacing `3` with `len(env_pkgs)`) not removing the rest of the `assert`...